src/chainparams.cpp

```diff
@@ -341,8 +341,8 @@ class CRegTestParams : public CChainParams {
 
 void CRegTestParams::UpdateActivationParametersFromArgs(const ArgsManager& args)
 {
-    if (gArgs.IsArgSet("-segwitheight")) {
-        int64_t height = gArgs.GetArg("-segwitheight", consensus.SegwitHeight);
+    if (args.IsArgSet("-segwitheight")) {
```

Comment:
This line had been added when local `args` already existed. I couldn't find any reason for the use of global variable, but maybe I'm missing something.